pkg/trojan/message.go

```diff
@@ -44,7 +44,7 @@ const (
 	TopicSize      = 32
 )
 
-var minerTimeout = 10 * time.Second
+var minerTimeout = 20 * time.Second
```

Comment:
??

Reply:
the CI is constantly timing out on the trojan tests (!!)